server.js

```diff
@@ -1,5 +1,69 @@
 // import your node modules
-
+const express = require('express')
```

Comment:
Try to be consistent with semi-colons. If you're going to omit them or keep them, stick with one. :)